nginx/manifest.json

```diff
@@ -40,7 +40,7 @@
     "dashboards": {
       "NGINX Plus base overview": "assets/dashboards/plus_overview.json",
       "NGINX-Metrics": "assets/dashboards/NGINX-Metrics_dashboard.json",
-      "NGINX-Overview": "assets/dashboards/NGINX-Overview_dashboard.json"
+      "NGINX-MSL-Overview": "assets/dashboards/NGINX-Overview-MSL_dashboard.json"
```

Comment:
I'd recommend not using the name `MSL` here. In general, it's an abstraction leak to append the type of dashboard to these shortnames; a dashboard is a dashboard so we should keep the short names descriptive of the content, not of the layout type. Also, renaming a short name here is a breaking change I believe.